docs/contributing.md

```diff
@@ -0,0 +1,73 @@
+# Contributing to Kro
```

Comment:
This looks like its duplicating the CONTRIBUTING content in multiple places.

I dont think we should be doing that. I also dont think we need 2x contributing.md, one at the top level and one in docs, so this instance of the file could I think be deleted.